Stellar-transaction.x

```diff
@@ -602,6 +603,22 @@ case BUMP_FOOTPRINT_EXPIRATION_UNIFORM:
     uint32 ledgersToExpire;
 };
 
+enum RestoreFootprintType
+{
+    RESTORE_FOOTPRINT_V1 = 0
```

Comment:
This is a little weird, but I think we should make the op a union in case we need to update how restoration works for evicted entries.

Reply:
Maybe just leave an extension point for now? It will achieve the same without much verbosity.

Reply:
I was thinking about that but wasn't sure if it'd work. If we use an extension point and then switch to a union later, can the type in the union be an enum?

Reply:
Actually I think you're right. It would still be binary compatible.

Reply:
IIUC yes (and this is why we define extension points at the top of the structs - that provides binary compatibility with unions.

Reply:
I updated both the bump and restore ops to use extension points.